features/facebookImport/src/model/analyses/ministories/connected-advertisers-analysis.js

```diff
@@ -1,19 +1,66 @@
+import React from "react";
+
+import i18n from "../../../i18n.js";
 import RootAnalysis from "./root-analysis.js";
 
+import ListOfDetails from "../../../components/listOfDetails/listOfDetails.jsx";
+
 export default class ConnectedAdvertisersAnalysis extends RootAnalysis {
     get title() {
-        return "Connected Advertisers";
+        return i18n.t("connectedAdvertisersMiniStory:title");
+    }
+
+    get label() {
+        return RootAnalysis.Labels.NONE;
```

Comment:
Sorry, what's this?

Reply:
Its in all active analyses, the default is having a "tech demo" label

Reply:
OK, I missed that.